GCMessengerSDKSample/GCMessengerSDKSample/ChatWrapperViewController.swift

```diff
@@ -53,14 +53,13 @@ class ChatWrapperViewController: UIViewController {
 
 extension ChatWrapperViewController: ChatControllerDelegate {
     func shouldPresentChatViewController(_ viewController: UINavigationController!) {
+        chatVC = viewController
```

Comment:
is it being cleared at some point? 
you hard referencing the ViewController - can it lead to memory leaks if it is not being released

Reply:
1. chatVC is a confusing name for something that doesn't represent a ViewController.
2. With regard to what Tehila wrote, why do we need to save the NavigationController that is being injected as a member?